src/theme.ts

```diff
@@ -1,79 +1,35 @@
 const colors = {
-  black: '#000000',
-  white: '#ffffff',
-
-  // Background
-  bg2: '#F9F9F9',
-  bg3: '#F8F6F6',
-  bg4: '#F4F4F4',
-
-  // Grey
-  grey2: '#ECECEC',
-  grey3: '#E5E5E5',
-  grey4: '#D2D2D2',
-  grey5: '#AEAEAE',
-  grey6: '#9E9E9E',
-  grey7: '#818181',
-  grey8: '#636768',
-
-  // Pink
-  pink2: '#FFB2B4',
-  pink3: '#FF8A8D',
-  pink4: '#FF7695',
-  pink5: '#FF4E75',
-  pink6: '#FD5064',
-  pink7: '#E0364A',
-  pink8: '#AB1D20',
-
-  // Green
-  green2: '',
-  green3: '#A7D6C6',
-  green4: '',
-  green5: '#55D6AA',
-  green6: '#4CB99F',
-  green7: '#469389',
-  green8: '#16716E',
-
-  // Blue
-  blue2: '#ACB6D8',
-  blue3: '#94A0C6',
-  blue4: '#6F7AA0',
-  blue5: '#545E7E',
-  blue6: '#363F59',
-  blue7: '#112035',
-  blue8: '#05090E',
+  // Core brand
+  primary: '#FF4E75',
+  secondary: '#112035',
+  white: '#FFFFFF',
```

Comment:
I think we should also keep the black just in case even if it's not in the brand guide might still be useful